ops/ccs-ops-misc/load_test/README.md

```diff
@@ -0,0 +1,155 @@
+to avoid caching affecting the response times of repeat runs.
+
+These tests are intended to be run from a server instance which has access permissions to reach out to the BFD endpoints. How to
+do this is outside the scope of this document, but instructions for this can be found in this more detailed run guide on confluence:
+https://confluence.cms.gov/display/BB2/Run+the+BFD+Load+Tests
+
+
+## Setup
+The tests use a few python libraries that will need to be installed on the run box prior to starting the tests. Below are the required
+libraries, what they do, and the command to install them.
+
+**Pyyaml**
+
+What: Library for reading/writing yaml files
+Why: Used to save the configuration values to a yaml file that can be used for future runs.
+Install: _pip3 install pyyaml_
+See: https://pypi.org/project/PyYAML/
+
+**Psycopg2**
+
+What: Library for connecting to a postGres database programatically via python.
+Why: The tests use this to connect to the environment's database in order to gather various randomized data sets to run each test.
+Install: _pip3 install psycopg2-binary_
+See: https://pypi.org/project/psycopg2/
+
+**Requests**
+
+What: Library for making http requests easily
+Why: Making calls to gather data for the coverage tests from the patient endpoint. 
+Offers convenience over python's default http request get().
+Install: _pip3 install requests_
+See: https://pypi.org/project/requests/
+
+
+In addition to these libraries, you'll also need to copy a PEM (credentials) file and set the user to your SSH username on the test box. 
+Run the following two commands on the box:
+
+```
+sudo cp /usr/local/bfd-server/bluebutton-backend-test-data-server-client-test-keypair.pem ~/
+
+sudo chown <SSH-username>:<SSH-username> bluebutton-backend-test-data-server-client-test-keypair.pem
+```
+
+Once this is done you'll have a modified PEM to use for the tests in your user directory.
+
+
+## Running a test
+
+There are a number of tests which each test various endpoints (and param combinations for those endpoints) for different released versions of BFD.
+
+The tests run headless, i.e. without the web GUI, because we'll be executing the tests over ssh in the terminal on a remote box.
+
+The tests are run with parameters that specify all the test params and test file to use, so the test can be run in a single line. This is primarily meant to allow support for running tests in Jenkins, but can also be used to automate test suites.
+
+A single-line test will look like this:
+
+_python3 runtests.py --homePath="~/" --clientCertPath="~/bluebutton-backend-test-data-server-client-test-keypair.pem" --databaseHost="**<AWS_DB_host>**" --databaseUsername="**<db_username_from_keybase>**" --databasePassword="**<db_password_from_keybase>**" --testHost="https://test.bfd.cms.gov" --testFile="./v1/eob_test.py" --testRunTime="1m" --maxClients="100" --clientsPerSecond="5"_
```

Comment:
The file ./v1/eob_test.py doesn't exist any longer, would be good to replace here with an existing test.

Reply:
Good call!